server.js

```diff
@@ -2,12 +2,10 @@ const express = require('express');
 
 const app = express();
 
+app.use(express.static('frontend'));
 // Middleware to parse JSON request body
-app.use(express.json());
+// app.use(express.json());
```

Comment:
can delete

Reply:
as per our chat, I will uncomment it, in case will break your json